hack/lib/serverless.bash

```diff
@@ -153,6 +144,11 @@ function deploy_knativeserving_cr {
 
   timeout 900 '[[ $(oc get knativeserving.operator.knative.dev knative-serving -n $SERVING_NAMESPACE -o=jsonpath="{.status.conditions[?(@.type==\"Ready\")].status}") != True ]]'  || return 7
 
+  # TODO: Remove this when the previous release has readiness checks available (since 1.12).
```

Comment:
This is not captured by readiness checks at all. If this is the namespace that's failing, we have a different bug.

Reply:
OK. Hmm...

Reply:
The failure was here: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-knative-serverless-operator-master-4.6-e2e-aws-ocp-46-continuous/1325770323794071552